src/main/java/com/greghaskins/spectrum/Box.java

```diff
@@ -0,0 +1,38 @@
+package com.greghaskins.spectrum;
+
+/**
+ * For use with nested steps inside a test, this box allows data passing between tests
+ */
+public class Box<T> {
```

Comment:
Spectrum used to have a similar structure (`Value`, see [example](https://github.com/greghaskins/spectrum/tree/0.7.1#example)), but I removed that in favor of `let()` to match the RSpec API. See #43. We should consider adding that back if you think it would be necessary, or at least use a consistent name/API.